lib/Doctrine/Common/Annotations/TokenParser.php

```diff
@@ -119,6 +134,52 @@ public function parseUseStatement()
         return $statements;
     }
 
+    private function parseUseStatementV8(): array
+    {
+        $groupRoot = '';
+        $class = '';
+        $alias = '';
+        $statements = [];
+        $explicitAlias = false;
+        while (($token = $this->next())) {
+            $isNameToken = $token[0] === T_STRING || $token[0] === T_NS_SEPARATOR;
+
+            if ($token[0] === T_NAME_QUALIFIED || $token[0] === T_NAME_FULLY_QUALIFIED) {
+                $class .= $token[1];
+
+                $classSplit = explode('\\', $token[1]);
+                $alias = $classSplit[count($classSplit) - 1];
+            } else if($token[0] === T_NS_SEPARATOR) {
```

Comment:
Do we still need all that handling in PHP 8 ? wouldn't use statements always be parsed as a single token and so no `T_NS_SEPARATOR` to deal with anymore ?

Reply:
Its there only when you have a group

```php
use xxx/{Test, Foo}
```